pkgs/top-level/all-packages.nix

```diff
@@ -1082,6 +1082,8 @@ with pkgs;
 
   mongosh = callPackage ../development/tools/mongosh { };
 
+  monophony = callPackage ../applications/audio/monophony { };
```

Comment:
This uses default `callPackage` with no args, and is exposed at the top-level, so it's better in `pkgs/by-name`. You can then remove this line.

Reply:
Oh, okay, thanks! Is this documented anywhere, i havent heard of by-name, so i don't know how to do it.

Reply:
it's documented in the readme in that folder

Reply:
Would i create a new folder called mo in by-name? and copy my monophony folder into there?

Reply:
yes, it'd also be `package.nix` instead of `default.nix`

Reply:
Thanks.

Reply:
Hmm, that didnt seem to work. Anything else i have to do? I currently get this when rebuilding: `error: attribute 'monophony' missing` at `inputs.nixpkgs-local.legacyPackages.${system}.monophony`

Reply:
Wait, i think i hadnt committed

Reply:
Yep, works now

Reply:
Pushed it